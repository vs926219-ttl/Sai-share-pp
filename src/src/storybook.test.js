import initStoryshots from '@storybook/addon-storyshots';
import { render } from '@testing-library/react';

// using React testing library renderer and serializer because of issues with popup snapshots
const reactTestingLibrarySerializer = {
  print: (val, serialize) => serialize(val.container.firstChild),
  // eslint-disable-next-line no-prototype-builtins
  test: val => val && val.hasOwnProperty('container'),
};

initStoryshots({
  renderer: render,
  snapshotSerializers: [reactTestingLibrarySerializer],
});
