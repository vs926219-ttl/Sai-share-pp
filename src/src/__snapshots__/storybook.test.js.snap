// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Back Button Default 1`] = `
<button
  class="MuiButtonBase-root MuiButton-root MuiButton-text button defaultButton backButton"
  tabindex="0"
  type="button"
>
  <span
    class="MuiButton-label"
  >
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root arrowBackIcon"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M20,11H7.83l5.59-5.59L12,4,4,12l8,8,1.41-1.41L7.83,13H20Z"
      />
    </svg>
    <span
      class="action"
      data-testid="ASN Drop"
    >
      ASN Drop
    </span>
  </span>
  <span
    class="MuiTouchRipple-root"
  />
</button>
`;

exports[`Storyshots Back Button Dynamic Breadcrumb Loadbody 1`] = `
<button
  class="MuiButtonBase-root MuiButton-root MuiButton-text button defaultButton backButton"
  tabindex="0"
  type="button"
>
  <span
    class="MuiButton-label"
  >
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root arrowBackIcon"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M20,11H7.83l5.59-5.59L12,4,4,12l8,8,1.41-1.41L7.83,13H20Z"
      />
    </svg>
    <span
      class="action"
      data-testid="ASN Drop"
    >
      ASN Drop
    </span>
  </span>
  <span
    class="MuiTouchRipple-root"
  />
</button>
`;

exports[`Storyshots Back Button Dynamic Breadcrumb Wiring Harness 1`] = `
<button
  class="MuiButtonBase-root MuiButton-root MuiButton-text button defaultButton backButton"
  tabindex="0"
  type="button"
>
  <span
    class="MuiButton-label"
  >
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root arrowBackIcon"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M20,11H7.83l5.59-5.59L12,4,4,12l8,8,1.41-1.41L7.83,13H20Z"
      />
    </svg>
    <span
      class="action"
      data-testid="ASN Drop"
    >
      ASN Drop
    </span>
  </span>
  <span
    class="MuiTouchRipple-root"
  />
</button>
`;

exports[`Storyshots Components/ Table Default 1`] = `
<div
  style="height: 800px; width: 800px; background-color: rgb(243, 245, 249); padding: 3em 10px;"
>
  <div
    class="baseContainer"
    style="width: 100%; height: 100%;"
  >
    <div
      class="basicTableContainer"
      style="width: 100%;"
    >
      <div
        style="overflow: visible; height: 0px; width: 0px;"
      >
        <div
          class="basicTable"
          style="height: 0px; width: 0px;"
        >
          <div>
            <div
              class="tableHeader scrollBar"
              data-rbd-droppable-context-id="0"
              data-rbd-droppable-id="droppable"
              style="height: 45px;"
            >
              <div
                data-rbd-draggable-context-id="0"
                data-rbd-draggable-id="_serialNumber"
                style="text-align: left; padding-left: 7px; padding-right: 2px; display: inline-block; width: 40px; background-color: rgb(255, 255, 255);"
              >
                <span
                  style="padding-bottom: 2px; display: inline-block;"
                >
                  #
                </span>
              </div>
              <div
                aria-describedby="rbd-hidden-text-0-hidden-text-0"
                data-rbd-drag-handle-context-id="0"
                data-rbd-drag-handle-draggable-id="asn"
                data-rbd-draggable-context-id="0"
                data-rbd-draggable-id="asn"
                draggable="false"
                role="button"
                style="text-align: left; display: inline-block; width: 120px; background-color: rgb(255, 255, 255);"
                tabindex="0"
              >
                <div
                  class="headerWithMenu"
                >
                  <div
                    class="ellipsisText"
                    style="text-align: left;"
                  >
                    ASN
                  </div>
                  <svg
                    aria-hidden="true"
                    aria-label="menu-for-column-ASN"
                    class="MuiSvgIcon-root filterIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"
                    />
                  </svg>
                </div>
                <div
                  style="background: rgb(250, 250, 250);"
                />
              </div>
              <div
                aria-describedby="rbd-hidden-text-0-hidden-text-0"
                data-rbd-drag-handle-context-id="0"
                data-rbd-drag-handle-draggable-id="asnStatus"
                data-rbd-draggable-context-id="0"
                data-rbd-draggable-id="asnStatus"
                draggable="false"
                role="button"
                style="text-align: left; display: inline-block; width: 150px; background-color: rgb(255, 255, 255);"
                tabindex="0"
              >
                <div
                  class="headerWithMenu"
                >
                  <div
                    class="ellipsisText"
                    style="text-align: left;"
                  >
                    VC Status
                  </div>
                  <svg
                    aria-hidden="true"
                    aria-label="menu-for-column-VC Status"
                    class="MuiSvgIcon-root filterIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"
                    />
                  </svg>
                </div>
                <div
                  style="background: rgb(250, 250, 250);"
                />
              </div>
              <div
                aria-describedby="rbd-hidden-text-0-hidden-text-0"
                data-rbd-drag-handle-context-id="0"
                data-rbd-drag-handle-draggable-id="material"
                data-rbd-draggable-context-id="0"
                data-rbd-draggable-id="material"
                draggable="false"
                role="button"
                style="text-align: left; display: inline-block; width: 120px; background-color: rgb(255, 255, 255);"
                tabindex="0"
              >
                <div
                  class="headerWithMenu"
                >
                  <div
                    class="ellipsisText"
                    style="text-align: left;"
                  >
                    Part
                  </div>
                  <svg
                    aria-hidden="true"
                    aria-label="menu-for-column-Part"
                    class="MuiSvgIcon-root filterIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"
                    />
                  </svg>
                </div>
                <div
                  style="background: rgb(250, 250, 250);"
                />
              </div>
              <div
                aria-describedby="rbd-hidden-text-0-hidden-text-0"
                data-rbd-drag-handle-context-id="0"
                data-rbd-drag-handle-draggable-id="materialStatus"
                data-rbd-draggable-context-id="0"
                data-rbd-draggable-id="materialStatus"
                draggable="false"
                role="button"
                style="text-align: left; display: inline-block; width: 120px; background-color: rgb(255, 255, 255);"
                tabindex="0"
              >
                <div
                  class="headerWithMenu"
                >
                  <div
                    class="ellipsisText"
                    style="text-align: left;"
                  >
                    Part Status
                  </div>
                  <svg
                    aria-hidden="true"
                    aria-label="menu-for-column-Part Status"
                    class="MuiSvgIcon-root filterIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"
                    />
                  </svg>
                </div>
                <div
                  style="background: rgb(250, 250, 250);"
                />
              </div>
            </div>
          </div>
          <div
            aria-label="grid"
            aria-readonly="true"
            class="ReactVirtualized__Grid ReactVirtualized__List scrollBar heightAdjustedListScroll"
            role="grid"
            style="box-sizing: border-box; direction: ltr; height: -47px; position: relative; width: 0px; will-change: transform; overflow-x: hidden; overflow-y: auto;"
            tabindex="0"
          />
        </div>
      </div>
      <div
        class="resize-triggers"
      >
        <div
          class="expand-trigger"
        >
          <div
            style="width: 1px; height: 1px;"
          />
        </div>
        <div
          class="contract-trigger"
        />
      </div>
    </div>
    <div
      class="actionsBar"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-text button tertiaryButton actionButton cutomizeTableButton"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label"
        >
          CUSTOMIZE TABLE
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
  <div
    class="copyAlert hidden"
  >
    Text copied
  </div>
</div>
`;

exports[`Storyshots Components/Search sort filter menu Default 1`] = `
<div>
   Header 
</div>
`;

exports[`Storyshots Components/Search sort filter menu Only Search 1`] = `
<div>
   Header 
</div>
`;

exports[`Storyshots Tab with ASN Count Default 1`] = `
<div
  class="tabWrapper notSelected"
  role="button"
  style="cursor: pointer; outline: none;"
  tabindex="0"
>
  <div
    class="countTitleWrapper"
  >
    <span
      aria-label="ASN Drop-count"
      class="notSelectedCount"
    >
      290
    </span>
    <span
      class="title"
    >
      ASN Drop
    </span>
  </div>
</div>
`;
